streamlit_app.py

```diff
@@ -26,16 +26,20 @@
     logo = Image.open("assets/grodecoder_logo.png")
     st.sidebar.image(logo)
     st.sidebar.markdown(
-        """
+        f"""
     **GroDecoder** extracts and identifies
     the molecular components of a structure file (PDB or GRO)
     issued from a molecular dynamics simulation.
 
     ---
-    [Source code](https://github.com/pierrepo/grodecoder)                 
+    [Source code](https://github.com/pierrepo/grodecoder)  
+
+    Last commit date: `{gd.get_git_last_commit_date()[1:-1]}`  
```

Comment:
Why this slice?

Reply:
Why remove the first and last characters?

Reply:
I see. It's due to the extra single quotes returned by `get_git_last_commit_date()` and `get_git_last_commit_hash()` functions.
In these functions, could you replace `.strip()` by `strip("' \n")` ? It should solve the need for the weird slice later.

Reply:
Done :white_check_mark: